src/components/PoolInfoCard.tsx

```diff
@@ -181,14 +174,19 @@ function PoolInfoCard({ data }: Props): ReactElement | null {
         <Box display="flex" flexWrap="wrap">
           {formattedData.tokens.map((token, index) => (
             <Box key={index} minWidth={{ xs: "100%", sm: "50%" }} mt={2}>
-              <Box display="flex" flexWrap="nowrap">
-                <img alt="icon" src={token.icon} width="24px" height="24px" />
+              <Box display="flex" flexWrap="nowrap" alignItems="center">
+                <TokenIcon
+                  alt="icon"
+                  symbol={token.symbol}
+                  width={20}
+                  height={20}
+                />
                 <Typography
                   variant="subtitle1"
                   ml={1}
                 >{`${token.symbol} ${token.percent}`}</Typography>
               </Box>
-              <Typography className="tokenValue">${token.value}</Typography>
+              <Typography data-testid="tokenValue">{token.value}</Typography>
```

Comment:
Can you make this more unique to the component? This is where we will start having clashing of data-testid names. 

Maybe something like `poolInfoCardTokenValue`

Reply:
Oh, right now this component, don't need testid actually. removed